website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,112 @@
+---
+layout: docs
+page_title: Enterprise to CE downgrade Process
+description: >-
+  Guide to downgrade from consul enterprise to consul ce
+---
+
+## Introduction
+
+This document describes some best practices that you should follow when
+downgrading from Consul Enterprise to Consul CE.
+
+## Download the CE Version
+
+First, download the binary for the CE version you want.
+
+<Tabs>
+<Tab heading="Binary">
+
+All current and past versions of the CE and Enterprise releases are
+available here:
+
+- https://releases.hashicorp.com/consul
+
+</Tab>
+<Tab heading="Docker">
+
+Docker containers are available at this location:
+
+- **CE:** https://hub.docker.com/r/hashicorp/consul
+
+</Tab>
+
+</Tabs>
+
+## Prepare for the Downgrade to CE
+
+**1.** Take a snapshot:
+
+```
+consul snapshot save backup.snap
+```
+
+You can inspect the snapshot to ensure if was successful with:
+
+```
+consul snapshot inspect backup.snap
+```
+
+Example output:
+
+```
+ID           2-1182-1542056499724
+Size         4115
+Index        1182
+Term         2
+Version      1
+```
+
+This will ensure you have a safe fallback option in case something goes wrong. Store
+this snapshot somewhere safe. More documentation on snapshot usage is available here:
+
+- [consul.io/commands/snapshot](/consul/commands/snapshot)
+- [Backup Consul Data and State tutorial](/consul/tutorials/production-deploy/backup-and-restore)
+
+**2.** Temporarily modify your Consul configuration so that its [log_level](/consul/docs/agent/config/cli-flags#_log_level)
+is set to `debug`. After doing this, issue the following command on your servers to
+reload the configuration:
+
+```
+consul reload
+```
+
+This change will give you more information to work with in the event something goes wrong.
+
+
+**3.** Remove references to tenanted services from service-resolver, service-splitter and service-router config entries.
+
```

Comment:
```suggestion
1. Temporarily modify your Consul configuration so that its [log_level](/consul/docs/agent/config/cli-flags#_log_level)
is set to `debug`. After doing this, issue the following command on your servers to
reload the configuration:

    ```
    consul reload
    ```

   This change gives you more information to work with in the event something goes wrong.


1. Remove references to services in non-default namespaces and partitions from service resolver, service splitter, and service router configuration entries.

```